libcontainer/container_linux.go

```diff
@@ -1109,6 +1118,17 @@ func (c *linuxContainer) refreshState() error {
 	return c.state.transition(&stoppedState{c: c})
 }
 
+func (c *linuxContainer) isProcessUnique(pid int) (bool, error) {
```

Comment:
`doesProcessExist` is probably a better name for this function.  Processes are unique, and with this function we're just doing a better (but still not perfect) job of identifying whether a given /proc entry is the container process we're interested in.